weaviate/collections/data.py

```diff
@@ -336,24 +366,24 @@ def insert_many(
 
     def replace(
         self,
-        properties: Properties,
         uuid: UUID,
-        vector: Optional[List[float]] = None,
+        properties: Properties,
         references: Optional[WeaviateReferences] = None,
+        vector: Optional[List[float]] = None,
```

Comment:
why did you change the order?

Reply:
I moved them into order of importance from the PoV of the `replace` method

Reply:
can you explain? I don't see why this order is better than the old one? This would be another breaking change and I think one that is harder to spot

Reply:
Yeah, given the number of other breaks we're doing I'll remove these ones